Docker.README.md

```diff
@@ -0,0 +1,41 @@
+# Supra DHT
```

Comment:
This is non blocking but we can also just call this supra, though we can talk about the DHT it is using etc...